test/rest-adapter.test.js

```diff
@@ -217,20 +217,20 @@ describe('RestAdapter', function() {
       });
     });
 
-    describe('getHttpMethod', function() {
+    describe('getEndpoints()[0].verb', function() {
       it('returns POST for `all`', function() {
         var method = givenRestStaticMethod({ http: { verb: 'all' }});
-        expect(method.getHttpMethod()).to.equal('POST');
+        expect(method.getEndpoints()[0].verb).to.equal('POST');
       });
 
       it('returns DELETE for `del`', function() {
         var method = givenRestStaticMethod({ http: { verb: 'del' }});
-        expect(method.getHttpMethod()).to.equal('DELETE');
+        expect(method.getEndpoints()[0].verb).to.equal('DELETE');
       });
 
       it('returns upper-case value otherwise', function() {
         var method = givenRestStaticMethod({ http: { verb: 'get' }});
-        expect(method.getHttpMethod()).to.equal('GET');
+        expect(method.getEndpoints()[0].verb).to.equal('GET');
       });
```

Comment:
>I am proposing the following:

>* Rework all tests using deprecated methods to use the new official methods
* In unit-tests that are testing specifically these deprecated methods, temporarily disable deprecation messages via https://github.com/dougwilson/nodejs-depd#processenvno_deprecation

Here I tried to use the new official methods in tests; for instance using `getEndpoints()[0].verb` instead of `getHttpMethod()` which is now deprecated.

However I believe what I did is not what you proposed. IIUC, here in this test that we are explicitly testing this deprecated method, I should just have them untouched, but without deprecation message. However, based on [other usages](https://github.com/strongloop/loopback/blob/6964914bab64496c789c434fec39d2914231ee23/Gruntfile.js#L9) I tried `process.env.NO_DEPRECATION = 'strong-remoting';` but still it shows the deprecation messages...